sys/riscv/include/cherireg.h

```diff
@@ -196,7 +196,7 @@
  * CHERI_BASELEN_BITS is used in cheribsdtest_cheriabi.c.  The others are
  * unused.
  */
-#define	CHERI_BASELEN_BITS	10
+#define	CHERI_BASELEN_BITS	11
 #define	CHERI_SEAL_BASELEN_BITS	5
```

Comment:
Unrelated but the SEAL one is wrong, they are both the same. I think we should remove the defines it and just rely on the rounding instructions.